test/views/juttle-view.spec.js

```diff
@@ -0,0 +1,24 @@
+require('chai').should();
+var JuttleView = require('../../src/views/juttle-view');
+var v = require('../../src/lib/object-validation');
```

Comment:
seems using the character `v` would be confusing for someone scanning the code (it is for me). I know we do this in a million places but perhaps would should start using something more descriptive?